tests/integrations/loguru/test_loguru.py

```diff
@@ -72,7 +73,8 @@ def test_just_log(
             (breadcrumb,) = breadcrumbs
             assert breadcrumb["level"] == expected_sentry_level
             assert breadcrumb["category"] == "tests.integrations.loguru.test_loguru"
-            assert breadcrumb["message"][23:] == formatted_message
+            # Check that the message matches the expected pattern with a line number
+            assert re.search(expected_pattern, breadcrumb["message"][23:])
```

Comment:
I believe we want to use [`fullmatch`](https://docs.python.org/3/library/re.html#re.fullmatch) rather than `search`. With `search`, the assertion passes if the string contains something matching the regex, but here, we instead want to ensure that the entire string matches